base/services/w32time/lang/ru-RU.rc

```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:     ReactOS W32Time Service
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Russian resource file
+ * TRANSLATORS: Copyright 2022 Nikki Next <nikki_next@yahoo.com>
+ */
+
+LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_DISPLAYNAME "Служба времени"
+    IDS_DESCRIPTION "Управляет синхронизацией даты и времени на всех клиентах и серверах в сети."
+END

```

Comment:
The file is formatted with Unix line endings aka "\n" and it seems your git configuration somehow ignores settings in `.gitattributes`. I will fix it for this PR, but please fix your configuration next time, because this results in always unstaged changes (line endings) on a properly configured git repo.